content/en/docs/ops/ambient/getting-started/gtwapi_test.sh

```diff
@@ -0,0 +1,89 @@
+#!/usr/bin/env bash
```

Comment:
Maybe a miscommunication, but only the @cleanup instructions need to be in this file. The bulk of the test should be included from test.sh, which has if checks around the parts that are different depending on gateway-api or not (as you had done previously).

Here's an example of what a typical gtwapi_test.sh looks like: https://github.com/istio/istio.io/blob/master/content/en/docs/tasks/traffic-management/ingress/ingress-sni-passthrough/gtwapi_test.sh